code/api/src/main/scala/com/avast/datadog4s/api/EventFactory.scala

```diff
@@ -0,0 +1,10 @@
+package com.avast.datadog4s.api
+
+import com.avast.datadog4s.api.event.Eventer
+
+trait EventFactory[F[_]] {
+  def eventer: Eventer[F]
```

Comment:
i'm not sure it makes sense to have `Eventer` here ... since it only has one method, we could probably add the `send` method to the `EventFactory` ... what do you think?

Reply:
also if we do that, we can probably just call it `EventSender` or something like that, since it's not really a factory anymore